packages/deploy-tooling/src/cli/index.ts

```diff
@@ -60,6 +60,9 @@ export function createCommand(name: 'deploy' | 'undeploy'): Command {
         command
             .option('--package <abap-package>', 'Package name for deploy target ABAP system')
             .option('--description <description>', 'Project description of the app')
+            .addOption(
+                new Option('--create-transport', 'Create transport request during deployment').conflicts(['transport'])
```

Comment:
The `createCommand` unit test can be extended to ensure the conflict is validated.

Reply:
Updated unit test